tests/unit_test/data_processor/upload_handler_log_processor_test.py

```
@@ -0,0 +1,144 @@
+        assert not log.get('exception')
+        assert log['start_timestamp']
+        assert log['collection_name']
+        assert not log.get('end_timestamp')
+        assert not log.get('Upload_errors')
+        assert log.get('file_name')
+        assert log['event_status'] == EVENT_STATUS.VALIDATING.value
+
+    def test_is_event_in_progress_true_raises(self):
+        bot="test_bot"
+        collection_name="test_collection"
+        assert UploadHandlerLogProcessor.is_event_in_progress(bot, collection_name, False)
```

Ruff (0.12.2)

Use of `assert` detected

(S101)